src/test/java/com/amihaiemil/docker/RtContainersTestCase.java

```diff
@@ -499,12 +499,48 @@ public void createsContainerWithId() throws Exception {
                     )
                 ),
                 URI.create("http://localhost/test"),
-                Mockito.mock(Docker.class)    
+                Mockito.mock(Docker.class)
             ).create(
                 Json.createObjectBuilder()
                     .add("Image", "ubuntu").build()
             ).getString("Id"),
             Matchers.is("df2419f4")
         );
     }
+
+    /**
+     * RtContainers.get() returns an RtContainer with the container id.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void getContainerWithIdStartsWithNotFound() throws Exception {
+        new ListedContainers(
+            new AssertRequest(
+                new Response(
+                    HttpStatus.SC_NO_CONTENT,
+                    "{ \"Id\": \"df2419f4\", \"Warnings\": [ ] }"
+                )
+            ),
+            URI.create("http://localhost/test"),
+            Mockito.mock(Docker.class)
+        ).get("df2419f4").start();
+    }
+
+    /**
+     * RtContainers.get() returns an RtContainer with the container id.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void getContainerWithIdStopWithNotFound() throws Exception {
+        new ListedContainers(
+            new AssertRequest(
+                new Response(
+                    HttpStatus.SC_NO_CONTENT,
+                    "{ \"Id\": \"df2419f4\", \"Warnings\": [ ] }"
+                )
+            ),
+            URI.create("http://localhost/test"),
+            Mockito.mock(Docker.class)
+        ).get("df2419f4").stop();
+    }
```

Comment:
@jackpan123 These 2 tests don't make sense. Why are you calling start and stop?

Just add a simple test only for the ``get(...)`` method and assert that a ``Container`` is being returned (not null) and that its JSON contains the ID.

Reply:
What `get(...)` method wants to express is that users cant directly obtain a certain container instead of iteratively searching. When the user finds it, who can directly manipulate the internal method of the `Container` .